### PR TITLE
[tools] Arrange tflitefile_tool files

### PR DESCRIPTION
**Pull request by YongseopKim**
Let's arrange tflitefile_tool before revision.

Signed-off-by: Yongseop Kim <yons.kim@samsung.com>

### CODE REVIEW

**Comment by YongseopKim** [comment]
for https://github.com/Samsung/ONE/issues/8086

File: tools/tflitefile_tool/model_parser.py [hunk 1]
```diff
@@ -14,129 +14,40 @@
...
-        nargs='*',
-        help="operator ID to print information (default: all)")
-    arg_parser.add_argument(
-        '-c',
-        '--config',
-        action='store_true',
-        help="Save the configuration file per operator")
-    arg_parser.add_argument(
-        '-p', '--prefix', help="file prefix to be saved (with -c/--config option)")
+    arg_parser.add_argument("input_file",
```
Comment (YongseopKim) [inline tools/tflitefile_tool/model_parser.py:28]: This change causes by formatter. Is it valid or not? I'm confused.

File: tools/tflitefile_tool/select_operator.py [hunk 2]
```diff
@@ -1013,8 +1013,9 @@ def GenerateOperator(new_builder, selected_operator, used_tensors_dic, used_opco
     builtin_option_type = selected_operator.BuiltinOptionsType()
```
Comment (YongseopKim) [inline tools/tflitefile_tool/select_operator.py:1013]: This file's changes cause by formatter.

**Comment by YongseopKim** [comment]
PTAL @chunseoklee and @hseok-oh

**Comment by YongseopKim** [comment]
there are something about formatter.

**Review by hseok-oh — APPROVED** [review]
LGTM :+1:

**Comment by YongseopKim** [comment]
FYI
```
~/Works/ONE/tools/tflitefile_tool(pr/tools/arrange_tflitefile_tool ✗) tree -L 2
.
├── ir
│   ├── graph_stats.py
│   ├── __init__.py
│   ├── operator_wrapping.py
│   └── tensor_wrapping.py
├── model_parser.py
├── parser
│   ├── __init__.py
│   ├── model_parser.py
│   └── operator_parser.py
├── printer
│   ├── __init__.py
│   ├── operator_printer.py
│   ├── option_printer.py
│   ├── subgraph_printer.py
│   └── tensor_printer.py
├── README.md
├── requirements.txt
├── saver
│   ├── config_saver.py
│   ├── __init__.py
│   └── model_saver.py
├── select_operator.py
└── tflite
    ├── AbsOptions.py
    ├── ...
```

**Review by chunseoklee — APPROVED** [review]

**Pull request merged**